nilearn/experimental/datasets/_datasets.py

```diff
@@ -0,0 +1,147 @@
+"""Fetching a few example datasets to use during development.
+
+eventually nilearn.datasets would be updated
+"""
+
+from __future__ import annotations
+
+from typing import Sequence
+
+from nilearn import datasets
+from nilearn.experimental.surface import _io
+from nilearn.experimental.surface._surface_image import (
+    FileMesh,
+    Mesh,
+    PolyMesh,
+    SurfaceImage,
+)
+
+
+def load_fsaverage(
+    mesh_name: str = "fsaverage5",
+) -> dict[str, PolyMesh]:
+    """Load fsaverage for both hemispheres."""
+    fsaverage = datasets.fetch_surf_fsaverage(mesh_name)
+    renaming = {
+        "pial": "pial",
+        "white": "white_matter",
+        "infl": "inflated",
+        "sphere": "sphere",
+        "flat": "flat",
+    }
+    meshes: dict[str, dict[str, Mesh]] = {
+        mesh_name: {
+            f"{hemisphere}": FileMesh(fsaverage[f"{mesh_type}_{hemisphere}"])
+            for hemisphere in ("left", "right")
+        }
+        for mesh_type, mesh_name in renaming.items()
+    }
+    renaming = {"curv": "curvature", "sulc": "sulcal", "thick": "thickness"}
```

Comment:
```suggestion
    renaming = {"curv": "curvature", "sulc": "sulcal", "thick": "thickness"}
```

that line should have been removed